src/controller.cpp

```diff
@@ -54,11 +56,15 @@ namespace bzzz
         int mFR = throttle + controlToPwmScaling * (-controls[0] + controls[1] - controls[2]);
         int mBL = throttle + controlToPwmScaling * (controls[0] - controls[1] - controls[2]);
         int mBR = throttle + controlToPwmScaling * (-controls[0] - controls[1] + controls[2]);
+        int mML = throttle + controlToPwmScaling * (-controls[0]              - controls[2]); // adding for new motor middle left
```

Comment:
The implementation needs to be completely different for quadcopters and hexacopters.